vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -732,6 +736,9 @@ components:
         tdx:
           type: boolean
           default: false
+        sev_snp:
+          type: boolean
+          default: false
```

Comment:
Don't you need host_data here?

Reply:
Nope, it's in the `PayloadConfig` struct, see https://github.com/cloud-hypervisor/cloud-hypervisor/pull/6965#issuecomment-2688746782 . Thanks